### PR TITLE
ci(gh-actions): Add GH Actions workflow

### LINKED ISSUE #46: Switch to GitHub Actions
Travis is pretty unreliable now so would like to convert to GitHub actions. Example: https://github.com/xdebug/vscode-php-debug/blob/main/.github/workflows/build.yml

### PR DESCRIPTION
**Pull request by weirdan**
Fixes felixfbecker/php-advanced-json-rpc#46

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+name: build
+
+on: [push, pull_request]
+
+env:
+  FORCE_COLOR: 1
+  GH_TOKEN: ${{ secrets.SEMANTIC_RELEASE_GITHUB_TOKEN }}
```
Comment (felixfbecker) [inline .github/workflows/build.yml:7]: Afaik GitHub automatically adds a secret "GITHUB_TOKEN" so it shouldn't be necessary to add one manually?
```suggestion
  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment (weirdan) [reply .github/workflows/build.yml:7]: That doesn't work with the version of `semantic-release` this repo is using, due the issues fixed in newer releases. @felixfbecker shall I bump node deps in this PR?

Comment (weirdan) [reply .github/workflows/build.yml:7]: @felixfbecker ping

Comment (weirdan) [reply .github/workflows/build.yml]: done

**Review by felixfbecker — APPROVED** [review]

**Comment by felixfbecker** [comment]
> As this may generate new releases it's probably advisable to disable semantic-release on Travis CI when this PR is merged

All you need to do for that I think is delete the travis.yml in the repo. Can you do that in this PR so Travis doesn't run after this PR?

**Comment by weirdan** [comment]
:tada:

**Pull request merged**